crypto/pkcs12/p12_mutl.c

```diff
@@ -187,7 +187,9 @@ int PKCS12_set_mac(PKCS12 *p12, const char *pass, int passlen,
     ASN1_OCTET_STRING *macoct;
 
     if (!md_type)
-        md_type = EVP_sha1();
+        md_type = EVP_sha256();
```

Comment:
Fetch instead of the legacy implementation?

Reply:
Possibly.  There is PR in progress #14474 making these changes.  @slontis, you close on that one?

Reply:
There is no point in doing fetch as the md_type is used only to get a NID in the PKCS12_setup_mac().

Reply:
I'll add a comment.